client/my-sites/checkout/composite-checkout/components/vat-form/index.tsx

```diff
@@ -225,6 +225,22 @@ export function VatForm( {
 					} }
 				/>
 			</div>
+			<div className="vat-form__row vat-form__row--full-width">
+				<Field
+					id={ section + '-address' }
+					type="text"
+					label={ String( translate( 'Address for VAT' ) ) }
+					value={ vatDetailsInForm.address ?? '' }
+					autoComplete="address"
+					disabled={ isDisabled }
+					onChange={ ( newValue: string ) => {
+						setVatDetailsInForm( {
+							...vatDetailsInForm,
+							address: newValue,
+						} );
+					} }
+				/>
+			</div>
 			{ savedVatDetails.id && (
 				<div>
 					<FormSettingExplanation>
```

Comment:
I actually didn't notice this when testing. Should it be above the Address field?

Reply:
🤔 That's an interesting question. I left it below the form because it felt better to me there but we could certainly move it. I'll wait for now because I know that @michaeldcain wants to do some styling of this form and it could be worked in there.